### PR TITLE
Adds link to bayesian inference

### LINKED ISSUE #12: Introduction to MCMC and Bayesian inference
In this proposed tutorial we plan to explain the following points:
 - What is a likelihood function
 - What is a prior
 - What is a posterior
 - What is an MCMC chain
 - How to use emcee to fit some function

Other things ?

### PR DESCRIPTION
**Pull request by EiffL**
This PR closes #12 .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,6 +65,11 @@ We always welcome new tutorial requests, just click [here](https://github.com/Co
    * *MRI reconstruction with TensorFlow*: [![youtube](https://img.shields.io/badge/-youtube-red?logo=youtube&labelColor=grey)](https://www.youtube.com/watch?v=HqNm5fowdTU) [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/CosmoStat/Tutorials/blob/tensorflow-tutorial/MRIReconstructionWithTensorflow.ipynb)  
    An illustration of how to use TensorFlow to solve a toy MRI reconstruction problem using classical iterative reconstructions, and a deep learning approach.
 
+7. **[Introduction to MCMC and Bayesian inference](https://github.com/zaccharieramzi/git-tuto)** |   [![youtube](https://img.shields.io/badge/-youtube-red?logo=youtube&labelColor=grey)](https://www.youtube.com/watch?v=EfWEGBHeA3k) [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/drive/1EUF7-CwamhAT6wedntStdb4x3dDbihoo?usp=sharing)
```
Comment (sfarrens) [inline README.md]: This is linking to Zach's Git tutorial.

**Review by sfarrens — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (EiffL) [reply README.md]: arf

Comment (EiffL) [reply README.md]: ok, replaced with a link to the notebook

**Review by sfarrens — APPROVED** [review]

**Comment by EiffL** [comment]
Thanks Sam!

**Comment by EiffL** [comment]
@all-contributors please add @EiffL for content

**Pull request merged**